test/fixtures/test-charmap.py

```diff
@@ -2,14 +2,21 @@
 import sys
 import locale
 
-reload(sys)
+try:
+  reload(sys)
+except NameError:
+  pass
+
 
 def main():
   encoding = locale.getdefaultlocale()[1]
   if not encoding:
     return False
 
-  sys.setdefaultencoding(encoding)
+  try:
+    sys.setdefaultencoding(encoding)
+  except NameError:
+    pass
   textmap = {
     'cp936': u'\u4e2d\u6587',
```

Comment:
Could we replace these `u'...'` strings by regular strings & use the `unicode_literals` future import instead?